adapters/beachfront/beachfront.go

```diff
@@ -348,12 +340,12 @@ func getBannerRequest(request *openrtb.BidRequest) (beachfrontBannerRequest, []e
 	bfBannerRequest.Domain = site.Domain
 
 	if request.User != nil {
-		if bfBannerRequest.User.ID == "" {
-			bfBannerRequest.User.ID = request.User.ID
+		if request.User.ID == "" {
+			request.User.ID = request.User.ID
```

Comment:
I believe you still want this line to be: `bfBannerRequest.User.ID = request.User.ID` and similar for L348